packages/lib-utils/src/k8s/hooks/k8s-watcher.ts

```diff
@@ -77,14 +77,14 @@ export const getReduxData = (immutableData, resource: WatchK8sResource) => {
 };
 
 export const getWatchData: GetWatchData = (resource, k8sModel, cluster) => {
-  if (!k8sModel || !resource || !resource.name || !resource.namespace) {
+  if (!k8sModel || !resource || !k8sModel.name || !resource.namespace) {
```

Comment:
@karelhala I think we modified this line to satisfy a Typescript error when migrating from console. This is what it is in console:
https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/k8s-watcher.ts#L74

```tsx
export const getIDAndDispatch: GetIDAndDispatch<SDKStoreState> = (resource, k8sModel, cluster) => {
  if (!k8sModel || !resource) {
    return null;
  }
``